1.10/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -20,6 +20,13 @@ In addition to the general [Docker requirements and recommendations for DC/OS][1
   mkfs -t xfs -n ftype=1 /dev/sdc1
   ```
 
+* If you are using Docker 1.12 or Docker 1.13, you must add `MountFlags=private` in `/etc/systemd/system/docker.service.d/execstart.conf` and then restart the Docker daemon with this command:
```

Comment:
Not mentioning at all sounds good to me. Too much alarming information in release notes (when a cause for them in unknown to developers) is not going to help customers at all.

I am hoping that this note will as a serve as a "lead" for people who are running docker 1.12.0 and docker 1.13.0, and they will make a decision based "lead" information have provided.